public/controlpanel.php

```diff
@@ -453,13 +453,11 @@ function ResetProgressForSelection() {
     <?php if ($permissions >= Permissions::Registered): ?>
         <div id="rightcontainer">
             <div class='component'>
-                <h3>Request Score Recalculation</h3>
-                <form method="post" action="/request/user/recalculate-score.php">
-                    <?= csrf_field() ?>
-                    <input type="hidden" name="user" value="<?= $user ?>">
-                    If you feel your score is inaccurate due to point values varying during achievement development, you can request a recalculation by using the button below.<br><br>
-                    <input value="Recalculate My Score" type='submit' size='37'>
-                </form>
+                <h3>Site Awards</h3>
+                <div style="margin-bottom: 10px">
+                    You can manually choose the order your earned awards (badges) are displayed.
+                </div>
+                <a class="btn btn-link" href="reorderSiteAwards.php">Reorder Site Awards</a>
```

Comment:
I've thought about that at first, but all the other buttons in `controlpanel.php` are for `<form>`s; i.e they perform immediate action on click and then refresh the page. This one's a link instead, it just takes the user to another page. I'd leave it as-is to not confuse the users (they might think "Reorder Site Awards" is an automated action otherwise).